src/name_bazaar/ui/pages/watched_names_page.cljs

```diff
@@ -81,24 +82,26 @@
                 :watched-name watched-item}])))]])))
 
 (defmethod page :route/watched-names []
-  [app-layout
-   [ui/Segment
-    {:class "watched-names"}
-    [:h1.ui.header.padded "Watched Names"]
-    [ui/Grid
-     {:class "layout-grid"}
-     [ui/GridRow
-      [ui/GridColumn
-       {:computer 8
-        :tablet 12
-        :mobile 16}
-       [add-watched-name-form]]]
-     [ui/GridRow
-      [ui/GridColumn
-       {:class :join-upper
-        :text-align :right}
-       [:a.clear-all
-        {:on-click #(dispatch [:watched-names/remove-all])}
-        "Clear All"]]]
-     [ui/GridRow
-      [watched-names-infinite-list]]]]])
+  [:div.wrapper
+   [name-bazaar-meta-tags]
```

Comment:
:title "NameBazaar - Watched Names"
:description "Watch status of your favourite ENS names"